src/app/components/Transactions/LogEvent.tsx

```diff
@@ -141,7 +198,7 @@ export const TransactionLogEvent: FC<{
           <>
             <dt>{t('transaction.header')}</dt>
             <dd>
-              <TransactionLink scope={scope} hash={event.tx_hash} />
+              <TransactionLink scope={scope} hash={transactionLinkValue || transaction.hash} />
```

Comment:
API now has event.eth_tx_hash! This can be simpler

Reply:
Right good point 👍